### PR TITLE
o-r integrations: apply sentence-style capitalization

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

o-r integrations: 
- apply sentence-style capitalization

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/plaato.markdown [hunk 1]
```diff
@@ -17,12 +17,12 @@ ha_integration_type: integration
 
 This integration sets up integration with [Plaato](https://www.plaato.io/) Airlock and Keg.
 
-### Plaato Airlock
+### Plaato airlock
```
Comment (klaasnicolaas) [inline source/_integrations/plaato.markdown]: It's a product name 🤔

File: source/_integrations/plaato.markdown [hunk 2]
```diff
@@ -17,12 +17,12 @@ ha_integration_type: integration
...
 
 A tool for beer brewers that wants a unique insight into the fermentation process.
 With this integration, you get all your fermentation data available in Home Assistant!
 
-### Plaato Keg
+### Plaato keg
```
Comment (klaasnicolaas) [inline source/_integrations/plaato.markdown]: Same

File: source/_integrations/plaato.markdown (hunk 1 above)
Comment (c0ffeeca7) [inline source/_integrations/plaato.markdown]:
```suggestion
### Plaato Airlock
```

File: source/_integrations/plaato.markdown (hunk 2 above)
Comment (c0ffeeca7) [inline source/_integrations/plaato.markdown]:
```suggestion
### Plaato Keg
```

**Review by klaasnicolaas — APPROVED** [review]
✅ Approved

**Comment by c0ffeeca7** [comment]
Thank you for the catch, @klaasnicolaas :+1:

**Pull request merged**